iris_grib/message.py

```diff
@@ -143,6 +143,9 @@ def data(self):
                 raise TranslationError(msg)
             if template in (20, 30, 90):
                 shape = (grid_section['Ny'], grid_section['Nx'])
+            elif template == 140:
+                shape = (grid_section['numberOfPointsAlongYAxis'],
+                         grid_section['numberOfPointsAlongXAxis'])
```

Comment:
Have you checked that we can't just use "Nx" and "Ny", and so integrate this into the section above ?

Reply:
noted, unfortunately when I tried this it didn't work hence adding these more complete names.